apps/studio/components/interfaces/Connect/pooler-icons.tsx

```diff
@@ -0,0 +1,360 @@
+'use client'
+
+import React, { useState, useEffect } from 'react'
```

Comment:
for consistency - import the necessary components from react, avoid importing `React` here